vrk-kpa/suomidigi
# SUOPA-526 Theme tag lists

To test: 
- `make fresh`
- Go to any Theme tag (Teema) and edit the taxonomy term
- Add **View** paragraphs to **Before content** paragraph field. See image below.
![image](https://user-images.githubusercontent.com/1712902/65606463-a1e83300-dfb3-11e9-845d-940371778cb2.png)
- Save the taxonomy term and view it.

**Note:** If the list is rather empty, there's just content missing. Create new content for each of the types and check if the list gets updated.
